### PR TITLE
Additional AVS entries for AVS[10-15] 

### PR DESCRIPTION
**Pull request by sblair01**
# Overview/Summary

Additional AVS entries for AVS[10-15] 

## This PR fixes/adds/changes/removes

1. Changes Service Health from medium to high
2. Add AVS 10-15 to address availability, networking, disaster recovery, and identity and access management
3. Update some learn more links with relevant descriptions
4. Address some typos and language in the existing APRL queries

### CODE REVIEW

**Comment by michielvanschaik** [comment]
@sblair01 can you also add the code files for AVS-10 to AVS-15 with "// under-development" in the kql?

**Review by ejhenry — APPROVED** [review]
LGTM

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 1]
```diff
@@ -257,4 +265,157 @@ Anyone with contributor access on the resource group hosting Azure VMware Soluti
...
+
+**Resource Graph Query/Scripts**
+
+{{< collapse title="Show/Hide Query/Script" >}}
+
+{{< code lang="sql" file="code/avs-10/avs-10.kql" >}} {{< /code >}}
+
+{{< /collapse >}}
+
+<br><be>
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:294]:
```suggestion
<br><br>
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 2]
```diff
@@ -257,4 +265,157 @@ Anyone with contributor access on the resource group hosting Azure VMware Soluti
...
+
+**Resource Graph Query/Scripts**
+
+{{< collapse title="Show/Hide Query/Script" >}}
+
+{{< code lang="sql" file="code/avs-11/avs-11.kql" >}} {{< /code >}}
+
+{{< /collapse >}}
+
+<br><be>
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:317]:
```suggestion
<br><br>
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 3]
```diff
@@ -257,4 +265,157 @@ Anyone with contributor access on the resource group hosting Azure VMware Soluti
...
+
+**Resource Graph Query/Scripts**
+
+{{< collapse title="Show/Hide Query/Script" >}}
+
+{{< code lang="sql" file="code/avs-12/avs-12.kql" >}} {{< /code >}}
+
+{{< /collapse >}}
+
+<br><be>
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:342]:
```suggestion
<br><br>
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 4]
```diff
@@ -257,4 +265,157 @@ Anyone with contributor access on the resource group hosting Azure VMware Soluti
...
+
+**Resource Graph Query/Scripts**
+
+{{< collapse title="Show/Hide Query/Script" >}}
+
+{{< code lang="sql" file="code/avs-14/avs-14.kql" >}} {{< /code >}}
+
+{{< /collapse >}}
+
+<br><be>
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:392]:
```suggestion
<br><br>
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 5]
```diff
@@ -122,7 +130,7 @@ To expand the vSAN datastore additional hosts can be added, up to the maximum su
...
+If a Multi-AZ deployment of Azure VMware Solution is required, needs an infrastructure SLA of 99.99%, or needs synchronous storage replication between AZs (RPO=0), then Azure VMware Solution Stretched Clusters should be considered. If you are in a region that supports stretched clusters, consider enabling this feature to spread the VMware vSAN datastore across two availability zones. Note: Configuring an Azure VMware Solution private cloud as a stretched cluster can only be done during initial implementation and requires twice the quota. This is due to a stretched cluster extending the cluster to the second availability zone.
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:133]:
```suggestion
If a Multi-AZ deployment of Azure VMware Solution is required, needs a financially backed SLA of 99.99%, or needs synchronous storage replication between AZs (RPO=0), then Azure VMware Solution Stretched Clusters should be considered. If you are in a region that supports stretched clusters, consider enabling this feature to spread the VMware vSAN datastore across two availability zones. Note: Configuring an Azure VMware Solution private cloud as a stretched cluster can only be done during initial implementation and requires twice the quota. This is due to a stretched cluster extending the cluster to the second availability zone.
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 6]
```diff
@@ -195,7 +203,7 @@ Ensure there are enough memory resources to avoid host resource exhaustion. Azur
 
 **Recommendation/Guidance**
 
-Alert when the cluster size of 14 hosts is reached. Periodically fire up alerts, to prompt the customer to plan for a new cluster or additional datastore, if growth driven solely by storage. Beyond 14 hosts, every time a new host is added, surface an alert.
+Alert when the cluster size of 14 hosts is reached. Periodically fire up alerts to plan for a new cluster or additional datastore if growth is driven solely by storage. Beyond 14 hosts, every time a new host is added, surface an alert.
```
Comment (oZakari) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:206]:
```suggestion
Alert when the cluster size of 14 hosts is reached. Additionally, periodic alerts should be set up to indicate when growth, especially driven by storage requirements, necessitates planning for a new cluster or the addition of extra datastores. Furthermore, beyond the threshold of 14 hosts, alerts should be triggered each time a new host is added to the cluster, allowing proactive monitoring and management of resource utilization.
```

**Review by oZakari — CHANGES_REQUESTED** [review]
Please review my suggestions and incorporate if you agreed.

**Comment by oZakari** [comment]
Closing this PR as requested by @michielvanschaik.

cc: @sblair01

**Pull request closed without merging**